League Sandbox Auto Setup/leagueSandboxAutoSetupForm.cs

```diff
@@ -36,11 +51,17 @@ private void OnAbortSuccessfully()
             startButton.Enabled = true;
             abortText.Visible = false;
         }
-
+        bool convertProjectsToX86 = false;
```

Comment:
I don't think you need it to be class variable. Also, if you indeed need it, please define it at the start of the class.